src/components/generic/Navbar/navbarItems.ts

```diff
@@ -15,10 +15,10 @@ interface INavbarItems {
   private: INavbarItem[];
 }
 
-export const getNavbarItems = (t: typeof useT, myOrg?: V2MonitoringOrganisationRead | null): INavbarItems => {
-  const visibility = Boolean(
-    myOrg && myOrg?.status !== "rejected" && myOrg?.status !== "draft" && myOrg.users_status !== "requested"
-  );
+export const getNavbarItems = (t: typeof useT, myOrg?: MyOrganisationConnection): INavbarItems => {
+  const { userStatus, organisation } = myOrg ?? {};
+  const { status } = organisation ?? {};
+  const visibility = Boolean(organisation && status !== "rejected" && status !== "draft" && userStatus !== "requested");
```

Comment:
It could be a good idea to create a separate file to store all the constants for terms like `rejected`, `requested`, etc., since I see they are repeated across components. That way, if we need to change one in the future, we can just update it in one place instead of modifying multiple components.

Reply:
In principal, I agree, at least for related components. For instance, all entities and their reports use the same underlying state machine, so their statuses are the same and if any of them changed, they would all change. These two are unique; organisation status would not necessarily change if the project status strings changed, so unifying them could actually cause the opposite effect of needing to break apart statuses that were kind of arbitrarily combined. 

In this case, I think that creating constants would just be to avoid "magic" strings, which is something that I used to do religiously in JS. However, the TS compiler will make sure we're only using strings that it believes are valid. 

In this case, `userStatus` is coming from the Organisation connection:
```
type OrganisationConnection = {
  organisation?: OrganisationDto;
};

type UserStatus = "approved" | "rejected" | "requested";
export type MyOrganisationConnection = OrganisationConnection & {
  organisationId?: string;
  userStatus?: UserStatus;
};
```

And the Organisation status is defined by the BE via the typings on the OrganisationDto
```
export type OrganisationDto = {
  status: "draft" | "pending" | "approved" | "rejected";
  name: string | null;
};
```

I imagine VS code does this as well, but in WebStorm, I get super handy hints about this stuff:
![image](https://github.com/user-attachments/assets/d9269d24-8c30-4673-b9fc-9ecb3de6e094)
![image](https://github.com/user-attachments/assets/eb5d038f-fa2a-44f6-bc0a-294b2b60493d)

Reply:
And here's an illustration of what happens in TS if we tried to compare against a misspelled string value: 
![image](https://github.com/user-attachments/assets/2a837a27-26ae-4710-88d7-607aed0a2155)

Reply:
In this case, it's best to leave it as is. I will approve it then.